spot_coreio_extension/update_bashrc

```diff
@@ -0,0 +1,5 @@
+# Adding all the necessary ros sourcing
+echo "" >> ~/.bashrc
+echo "## ROS" >> ~/.bashrc
+echo "source /opt/ros/$ROS_DISTRO/setup.bash" >> ~/.bashrc
+echo "source $SPOT_WS/install/setup.bash" >> ~/.bashrc
```

Comment:
maybe this is a naive question about Docker.  But what if the install is run twice?  Then these lines will be put in the `.bashrc` twice.  can we put some smarts in the script to grep for these lines and skip adding them again if they are already in there?

Reply:
By install are you referring to building the docker image, installing the extension on the CORE I/O or something different? 

When building the docker image this will only get copied once, but `source /opt/ros/$ROS_DISTRO/setup.bash` could be a duplicate if the base image already has this in the `.bashrc` 

The installation in the CORE I/O will load the docker image and run containers of that image. These steps will not modify the `.bashrc` of the docker image. So no issue of duplication there. 

However, I think this may be redundant. The docker image also defines the entry point which sources the same directories. Will test if this is true. 
https://github.com/bdaiinstitute/spot_ros2/blob/b067b7c0d21b2fb874cd1b7ab28dd7ff29264727/spot_coreio_extension/Dockerfile.arm64#L58

Reply:
Delete the `update_basrch` step not required.